.github/workflows/build-container.yml

```
@@ -32,6 +35,18 @@ jobs:
       - name: Checkout source
         uses: actions/checkout@v4
 
+      - name: Make script executable
+        if: inputs.is_frontend == 'true'
```

```suggestion
        if: inputs.is_frontend
```

Think this would suffice as it is actually a bool. The reason it's like this in some places in the fingertips workflow is because some of the outputs are exported to the GITHUB_OUTPUT as strings.